server/model.py

```diff
@@ -9,10 +8,34 @@ class Users(db.Model):
     email = db.Column(db.String(64), unique=True, index=True)
     username = db.Column(db.String(64), unique=True, index=True)
     password_hash = db.Column(db.String(128))
+    eventType = db.relationship("EventTypes", backref="user")
 
     def __init__(self, name, email, username, password_hash):
         self.name = name
         self.email = email
         self.username = username
         self.password_hash = password_hash
 
+    def __repr__(self):
+        return f"User - id:{self.id}, name:{self.name}, " \
+               f"email:{self.email}, username:{self.username}"
+
+class Appointments(db.Model):
```

Comment:
this is good for now. we might have to make changes once we get to the stage of making appointments